frontend/src/lib/icons/HogIcon.tsx

```diff
@@ -1,86 +1,34 @@
-import * as React from 'react'
+import React, { CSSProperties } from 'react'
```

Comment:
I don't have anything against these changes to `HogIcon` per se, but I'd like to add some context. The component was created with the following goals:

- Base class for everything logo-related in the interface, should be wrapped further if needed.
- Drop-in replacement for an antd icon
- Can be with or without the "PostHog" logo (the plan was to abstract this further into a `<HogLogo />` and use that instead of the logo svg from /frontend/public)
- Ability to change the colors at random, so we could animate it at will
- Ability to add a `className` for extra animation (e.g. set some css transitions)

I actually wanted to expand this and also make an animation where these three colored stripes move up and down in a wavy pattern, while the hedgehog moves to the right, making it look like he's walking.

![image](https://user-images.githubusercontent.com/53387/89280532-08fd5c00-d649-11ea-9409-6095d6468752.png)

The idea was to try if this could be a nice extra "on hover" animation for the toolbar button, which should also start using this icon as well instead of the HogLogo that it now uses.

I admit that the naming of the component doesn't hint at these "future goals".

Reply:
Ah, I see how it could be extended.
I believe we shouldn't use our logo as an antd icon though – icons illustrate a UX point, while the logo represents our brand. It should generally only be present as an icon in the website header, possibly footer. And maybe as large scale backdrop, like in the Launch Toolbar modal.
Logo colors also generally shouldn't be manipulated. As in either blue-red-yellow-black or grayscale. Animated stripes would be cool though.

Reply:
We could take this thought experiment further and replace the logo on the toolbar button with rocket as well then :). 
The look of the button was what this "logo icon" was trying to replicate. I also tried with a white background + border to make it look even more like the toolbar button, but that didn't look good.

(To be clear, I have nothing against the rocket and what you describe with icons vs logos makes total sense!)

Re: the colors, I didn't really think of making a green hedgehog (thought it might be cool on st patrick's day). The idea was more to quickly rotate the colors (blue replaces yellow, yellow replaces red, etc) and try another "hover" animation. Though, something like this should be done in CSS and not with react props.

I'm not sure how strict we are with the brand though. In all official communication of course... but I think the hedgehog is a playful character that wouldn't mind.

Reply:
Well, yeah, I'm not really sure how strict we are either. We should have a posthog.com/branding page for that, with guidelines plus materials in SVG/PNG/PDF (for our and media use). Some strictness might be good until then though.

How about this? I think it would actually look good, since it'd clearly represent just the toolbar and wouldn't look like just an extra logo.
![Zrzut ekranu 2020-08-4 o 12 36 56](https://user-images.githubusercontent.com/4550621/89284916-c3905d00-d64f-11ea-98be-1ade9612dd9f.png)

Reply:
That does indeed clearly show what you're getting into!

The only things is that it makes this part of the sidebar looks rather unaligned:

![image](https://user-images.githubusercontent.com/53387/89285393-8c6e7b80-d650-11ea-918b-f0219de39b6d.png)

I don't know if there's a good way to make this look nicer. I have two left hands when it comes to design :).

Reply:
Yeah. Actually it's misaligned in the current form (color logo without background) as well. It's just hard to see because the colors blend into the dark background. That's why I thought we should just stick to antd icons all the way in the sidebar.